### PR TITLE
Add VB.NET signatures

### LINKED ISSUE #3720: VB isn't displayed in language selector
In this article, <https://docs.microsoft.com/en-us/dotnet/api/system.web.ui.scriptmanager.registerclientscriptblock?view=netframework-4.8>, The VB language isn't shown in the selector on top of the page.

Related to https://github.com/MicrosoftDocs/feedback/issues/2309

### PR DESCRIPTION
**Pull request by Youssef1313**
@mairaw, May you check in the staging site if this solves https://github.com/dotnet/dotnet-api-docs/issues/3720 ?

Fixes #3720

### CODE REVIEW

**Comment by gewarren** [comment]
> @mairaw, May you check in the staging site if this solves #3720 ?

Yes, it does.

image

File: xml/System.Web.UI/ScriptManager.xml [hunk 1]
```diff
@@ -2676,6 +2687,7 @@ The registered control does not implement the <see cref="T:System.Web.UI.INaming
       <MemberSignature Language="C#" Value="public void RegisterDataItem (System.Web.UI.Control control, string dataItem, bool isJsonSerialized);" />
       <MemberSignature Language="ILAsm" Value=".method public hidebysig instance void RegisterDataItem(class System.Web.UI.Control control, string dataItem, bool isJsonSerialized) cil managed" />
       <MemberSignature Language="DocId" Value="M:System.Web.UI.ScriptManager.RegisterDataItem(System.Web.UI.Control,System.String,System.Boolean)" />
+      <MemberSignature Language="VB.NET" Value="Public Soid RegisterDataItem (control As System.Web.UI.Control, dataItem As String, isJsonSerialized As Boolean)" />
```
Comment (gewarren) [inline xml/System.Web.UI/ScriptManager.xml]:
```suggestion
      <MemberSignature Language="VB.NET" Value="Public Sub RegisterDataItem (control As System.Web.UI.Control, dataItem As String, isJsonSerialized As Boolean)" />
```

File: xml/System.Web.UI/ScriptManager.xml [hunk 2]
```diff
@@ -3088,6 +3106,7 @@ The registered control does not implement the <see cref="T:System.Web.UI.INaming
       <MemberSignature Language="C#" Value="public static void RegisterOnSubmitStatement (System.Web.UI.Control control, Type type, string key, string script);" />
       <MemberSignature Language="ILAsm" Value=".method public static hidebysig void RegisterOnSubmitStatement(class System.Web.UI.Control control, class System.Type type, string key, string script) cil managed" />
       <MemberSignature Language="DocId" Value="M:System.Web.UI.ScriptManager.RegisterOnSubmitStatement(System.Web.UI.Control,System.Type,System.String,System.String)" />
+      <MemberSignature Language="VB.NET" Value="Public Shared Sub RegisterOnSubmitStatement (control As System.Web.UI.Control, type As Type, key As String, script As String);" />
```
Comment (gewarren) [inline xml/System.Web.UI/ScriptManager.xml]:
```suggestion
      <MemberSignature Language="VB.NET" Value="Public Shared Sub RegisterOnSubmitStatement (control As System.Web.UI.Control, type As Type, key As String, script As String)" />
```

**Comment by mairaw** [comment]
Just FYI this will be undone every time we run CI since this information is automatically generated. This issue will need to be fixed: https://github.com/mono/api-doc-tools/issues/356

**Comment by Youssef1313** [comment]
Is the reason of the missing information known ?

**Comment by mairaw** [comment]
I don't know @Youssef1313. This would be a question for @joelmartinez.

**Pull request merged**